filters/include/pcl/filters/functor_filter.h

```diff
@@ -76,22 +77,40 @@ class FunctorFilter : public FilterIndices<PointT> {
    */
   void
   applyFilter(Indices& indices) override
+  {
+    applyFilter(executor::best_fit{}, indices);
+  }
+
+  template <typename Executor, typename std::enable_if_t<executor::is_instance_of_base_v<executor::inline_executor, Executor> || executor::is_instance_of_base_v<executor::omp_executor, Executor>, int> = 0>
```

Comment:
Hacky currently, will come up with better solution

Reply:
if you could make the check for more than a single executor type it would substantially reduce this line